deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -538,8 +538,8 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.18.1
+  replaces: jaeger-operator.v1.20.0
```

Comment:
This should be v1.19.0

Reply:
Fixed by re-running the sed command mentioned in the other comment.